label_studio/data_manager/actions/basic.py

```diff
@@ -89,7 +97,7 @@ def delete_tasks_predictions(project, queryset, **kwargs):
         'permissions': 'can_manage_annotations',
         'dialog': {
             'text': 'Send the selected tasks to all ML backends connected to the project.'
-                    'This operation migth be abruptly interrupted due to a timeout. ' 
+                    'This operation migth be abruptly interrupted due to a timeout. '
```

Comment:
```suggestion
                    'This operation might be abruptly interrupted due to a timeout. '
```
not your code but you touched it so here's a typo fix :)